bedrock/firefox/views.py

```diff
@@ -176,6 +191,33 @@ def sign_attribution_codes(codes):
     return {"attribution_code": code.decode(), "attribution_sig": sig}
 
 
+@require_POST
+@csrf_exempt
+def sms_send_to_device_ajax(request):
+    form = SMSSendToDeviceForm(request.POST)
+    if not form.is_valid():
+        return JsonResponse({"success": False, "errors": form.errors["phone_number"]})
+
+    to_number = form.cleaned_data["phone_number"]
+    # platform = form.cleaned_data["platform"]
+
+    # TODO: Update `sms_body` with actual content. May differ based on platform.
+    sms_body = "test"  # Limited to 1600 characters.
```

Comment:
Wow 1600 chars for the SMS limit? I guess they'll split them for us - but if they do, will it not break a URL, for instance?